build/manifest/main.go

```diff
@@ -62,6 +111,30 @@ func findManifest() (*model.Manifest, error) {
 		return nil, errors.Wrap(err, "failed to parse manifest")
 	}
 
+	// Update the manifest based on the state of the current commit
+	// Use the first version we find (to prevent causing errors)
+	var version string
+	tags := strings.Fields(BuildTagCurrent)
+	for _, t := range tags {
+		if strings.HasPrefix(t, "v") {
+			version = t
+			break
+		}
+	}
+	if version == "" {
+		if BuildTagLatest != "" {
+			version = BuildTagLatest + "+" + BuildHashShort
+		} else {
+			version = "v0.0.0+" + BuildHashShort
+		}
+	}
+	manifest.Version = strings.TrimPrefix(version, "v")
+
+	// Update the release notes url to point at the latest tag, if present.
+	if BuildTagLatest != "" {
+		manifest.ReleaseNotesURL = manifest.HomepageURL + "releases/tag/" + BuildTagLatest
+	}
+
```

Comment:
LGTM :+1:

If I understand correctly:
- if the current commit matches a tag, then we use that
- If not, use the latest tag and append the latest commit hash
- If no tag available, use `v0.0.0` and append the latest commit hash

Reply:
💯